pages/sketch/[[...roomId]].js

```diff
@@ -0,0 +1,300 @@
+            else setValidLink(false);
+            setSketchLink(e.target.value);
+        };
+
+        const handleSubmit = async (e) => {
+            setLoading(true);
+            e.preventDefault();
+            await createSketchRoom(sketchLink, sketchName);
+            callbackDone && callbackDone();
+            setLoading(false);
+
+        };
+
+        return (
+            <Form onSubmit={handleSubmit}>
+                <input type="text" placeholder={t('sketch name')} value={sketchName} onChange={(e) => setSketchName(e.target.value)} />
+                <input type="text" placeholder={t('link to sketch')} value={sketchLink} onChange={handleExistingSketch} />
+                { !validLink && sketchLink !=='' && <span>{ t('Make sure your link includes') }:  { getConfig().publicRuntimeConfig.authProviders.sketch.baseUrl }</span> }
+
+                <button type="submit" disabled={!sketchName || !validLink || loading}>{ loading ? <LoadingSpinnerInline inverted /> : t('Add existing sketch') }</button>
+                { errorMessage && <ErrorMessage>{ errorMessage }</ErrorMessage> }
+            </Form>);
+    };
+
+    if (!serviceSpaceId) return <LoadingSpinner />;
+
+    return (
+        <>
+            <IframeLayout.Sidebar>
+                <ServiceSubmenu
+                    title={<h2>/sketch</h2>}
+                    subheadline={t('What would you like to do?')}
+                    items={[
+                        { value: 'existingSketch', actionComponentToRender: ActionExistingSketch, label: t('Add existing sketch') },
+                        { value: 'newSketch', actionComponentToRender: ActionNewSketch, label: t('Create sketch') },
+                    ]}
+                />
+                { syncingServerSketches ?
+                    <LoadingSpinner /> :
+                    <>
+                    <ServiceTable>
+                        { matrix.spaces.get(serviceSpaceId).children?.map(roomId => {
+                            return <SketchLinkEntry roomId={roomId} key={roomId} />;
+                        })}
+                    </ServiceTable>
+                        {isSketchServerDown && <ErrorMessage>{t('Can\'t connect with the provided /sketch server. Please try again later.')}</ErrorMessage>}
+                </>
+
+                }
+            </IframeLayout.Sidebar>
+            { roomId && content && (
+                <IframeLayout.IframeWrapper>
+                    <IframeLayout.IframeHeader>
+                        <h2>{ matrix.rooms.get(roomId).name }</h2>
+                        <IframeLayout.IframeHeaderButtonWrapper>
+                            <button title={t('Copy pad link to clipboard')} onClick={copyToClipboard}>
+                                <Clipboard fill="var(--color-foreground)" />
+                            </button>
+                            <button title={t('Remove sketch from my library')} onClick={removeLink}>
```

Comment:
I remember for pads we used "Remove from library" to verbalize that these pads are not actually being "deleted" from the Etherpads server, but only removed from your Matrix account by deleting the Matrix room.
But for Spacedeck sketches we **do actually** delete the sketch, right? Therefore I'd go with "Delete sketch" here instead.

What do you think?

Reply:
yeah that makes sense, maybe we should actually give people the option to do both. So they can remove a sketch from their sketch folder without deleting it (if its a shared sketch for example). 
I _think_ this is addressed in [roomActions](https://github.com/medienhaus/medienhaus-spaces/tree/roomActions ). Or at least the difference between deleting and removing, visually we need to make this clear though as well and maybe stop using the bin icon for removing and only for deleting?